internal/host/nextsteprunnercmd.go

```diff
@@ -0,0 +1,44 @@
+package host
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/openshift/assisted-service/internal/common"
+)
+
+type NextStepRunnerConfig struct {
+	ServiceBaseURL       string
+	ClusterID            string
+	HostID               string
+	UseCustomCACert      bool
+	NextStepRunnerImage  string
+	SkipCertVerification bool
+}
+
+func GetNextStepRunnerCommand(config *NextStepRunnerConfig) (string, *[]string) {
```

Comment:
is it tested somewhere?

Reply:
I couldn't think of meaningful unit-tests :( It's just formatting. I'll try to come up with something useful.

Reply:
How about subsystem? 
Anything that make sure that this logic will not break.